client/cli/go/cmds/device.go

```diff
@@ -322,30 +324,83 @@ var deviceDisableCommand = &cobra.Command{
 }
 
 var deviceResyncCommand = &cobra.Command{
-	Use:     "resync [device_id]",
+	Use:     "resync [object_id]",
 	Short:   "Resync storage information about the device with operation system",
 	Long:    "Resync storage information about the device with operation system",
-	Example: "  $ heketi-cli device resync 886a86a868711bef83001",
+	Example: `  $ heketi-cli device resync 886a86a868711bef83001 `,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		//ensure proper number of args
 		s := cmd.Flags().Args()
+
+		//ensure proper number of args
 		if len(s) < 1 {
-			return errors.New("device id missing")
+			return errors.New("object/cluster id missing")
 		}
 
-		// Set node id
-		deviceId := cmd.Flags().Arg(0)
+		// set objectId
+		objectId := cmd.Flags().Arg(0)
+		// determine operation mode: CLUSTER od DEVICE
+		resyncCluster, err := cmd.Flags().GetBool("cluster")
+		if err != nil {
+			return err
+		}
 
 		// Create a client
 		heketi, err := newHeketiClient()
 		if err != nil {
 			return err
 		}
 
-		//set url
-		err = heketi.DeviceResync(deviceId)
-		if err == nil {
-			fmt.Fprintf(stdout, "Device %v updated\n", deviceId)
+		// a single device Id was specified and no cluster resync was requested
+		if resyncCluster == false {
+			fmt.Println("No cluster-wide resync operation requested. Using object_id as device_id.")
+			//set url
+			err = heketi.DeviceResync(objectId)
+			if err == nil {
+				fmt.Fprintf(stdout, "Device %v updated\n", objectId)
+			}
+		} else { // requested cluster resync..
+			// RESYNC ALL
+			fmt.Println("Cluster-wide resync operation requested. Using object_id as cluster_id.")
```

Comment:
I like the first sentence, nice and clear. I don't the 2nd half is needed.

Reply:
Another stray comment...
Definitely redundant, I'll remove the second line.